org-code-javabuilder/playground/src/main/java/org/code/playground/Board.java

```diff
@@ -0,0 +1,93 @@
+package org.code.playground;
+
+import java.io.FileNotFoundException;
+import org.code.media.Color;
+import org.code.media.Font;
+import org.code.media.FontStyle;
+
+public class Board {
+    /**
+     * Returns the width of the board. This will always be 400.
+     * 
+     * @return the width of the board in pixels.
+     */
+    public int getWidth();
+
+    /**
+     * Returns the height of the board. This will always be 400.
+     * 
+     * @return the height of the board in pixels.
+     */
+    public int getHeight();
+
+    /**
+     * Sets the background of the board to the provided image. The image will
+     * be scaled to fit the full board, which may distort the image if
+     * it is not square.
+     * 
+     * @param filename the name of the file from the asset manager to put in the
+     *                 background
+     * @throws FileNotFoundException if the file cannot be found in the asset
+     *                               manager
+     */
+    public void setBackgroundImage(String filename) throws FileNotFoundException;
+
+    /**
+     * Adds a clickable item to the board.
+     * 
+     * @param item the item to add. If the item is already on the board,
+     *             this method does nothing.
+     */
+    public void addClickableItem(ClickableItem item);
```

Comment:
Not sure what level of nitiness we're going for in these API reviews, but the argument should be `ClickableImage` I think? Looks like we've been waffling between `ClickableImage` and `ClickableItem`, not sure which name is final?

Reply:
Fixed! Should be `ClickableImage`.